src/models/notification.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+const { Model } = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+    class Notification extends Model {
+        static associate(models) {}
+    }
+    Notification.init(
```

Comment:
@Alexjes98 shouldn't the notifications include the recipients?

Reply:
Juan said that for the moment the notifications should ve delivered to every user, because there are no roles yet, but I think Pedro is working on that. I'm any case we should discuss that.

Reply:
@Alexjes98 even without no roles and if its delivery to all user it should be a way to address that I mean should be able to save notifications with a list of user that need to be notified if in that list are all the user is ok.
my concerns is if we dont do that this task would be incomplete and we might going to need another task for that later so if we cover that in advance when @PedroLabrador figure it out which user should be notified @PedroLabrador can just call this function to create the notifications